pkg/reconciler/intevents/pullsubscription/keda/pullsubscription_test.go

```diff
@@ -1108,6 +1108,7 @@ func newReceiveAdapter(ctx context.Context, image string, transformer *apis.URL)
 		SubscriptionID:   testSubscriptionID,
 		SinkURI:          sinkURI,
 		TransformerURI:   transformer,
+		AuthType:         "secret",
```

Comment:
It would make the code more maintainable and readable (easy access to GoDoc) if this used the constants defined in `authtype` instead of string literals. This comment applies to all the similar uses.